### PR TITLE
chore: mark version v3.0.0

### PR DESCRIPTION
**Pull request by hanselfmu**

### CODE REVIEW

**Comment by hanselfmu** [comment]
@mathiasbynens My original PR was mysteriously deleted by Github after I have reset release-v3.0.0 branch with master, hence the new PR.

File: package.json [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-  "version": "2.1.1-post",
+  "version": "3.0.0",
   "description": "A high-level API to control headless Chrome over the DevTools Protocol",
   "main": "index.js",
   "repository": "github:puppeteer/puppeteer",
   "engines": {
     "node": ">=10.18.1"
   },
   "puppeteer": {
-    "chromium_revision": "722234",
```
Comment (mathiasbynens) [inline package.json]: Please do this change (+ any test expectation updates or other Puppeteer changes needed to support the new Chromium) in a separate CL.

The Chromium roll and the Puppeteer version bump should be separate commits.

File: package.json [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+    "chromium_revision": "737027",
```
Comment (mathiasbynens) [inline package.json]:
```suggestion
    "chromium_revision": "722234",
```

**Review by mathiasbynens — APPROVED** [review]
LGTM but let's get https://github.com/puppeteer/puppeteer/pull/5644 landed first.

**Review by mathiasbynens — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -301,7 +301,7 @@ See [Contributing](https://github.com/puppeteer/puppeteer/blob/master/CONTRIBUTI
 Historically, Puppeteer supported Firefox indirectly through puppeteer-firefox, which relied on a custom, patched version of Firefox. This approach was also known as “Juggler”.
 After discussions with Mozilla, we collectively concluded that relying on custom patches was infeasible.
 Since then, we have been collaborating with Mozilla on supporting Puppeteer on “stock” Firefox.
-From Puppeteer v2.1.0 onwards, as an experimental feature, you can specify [`puppeteer.launch({product: 'firefox'})`](https://github.com/puppeteer/puppeteer/blob/v2.1.1/docs/api.md#puppeteerlaunchoptions) to run your Puppeteer scripts in Firefox Nightly, without any additional custom patches.
+From Puppeteer v2.1.0 onwards, as an experimental feature, you can specify [`puppeteer.launch({product: 'firefox'})`](https://github.com/puppeteer/puppeteer/blob/v3.0.0/docs/api.md#puppeteerlaunchoptions) to run your Puppeteer scripts in Firefox Nightly, without any additional custom patches.
```
Comment (mathiasbynens) [inline README.md]:
```suggestion
From Puppeteer v2.1.0 onwards, as an experimental feature, you can specify [`puppeteer.launch({product: 'firefox'})`](https://github.com/puppeteer/puppeteer/blob/v2.1.0/docs/api.md#puppeteerlaunchoptions) to run your Puppeteer scripts in Firefox Nightly, without any additional custom patches.
```
